python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -486,6 +486,11 @@ class PythonFunctionValue extends FunctionValue {
         )
     }
 
+    /** Gets a control flow node corresponding to the value of a return statement */
```

Comment:
I know this is what the old text said, but perhaps it would be good to specify that the return statement in question belongs to the `FunctionValue`?

Reply:
alrighty then, does it look good now? :smile: